phase1/openstack/openstack.jsonnet

```diff
@@ -49,6 +49,7 @@ function(cfg)
         password: openstack.password,
         auth_url: openstack.auth_url,
         insecure: openstack.insecure,
+        cacert_file: "{$file(\%s\")} % openstack.os_cacert,
```

Comment:
This seems fine, since it looks like we can specify either the content or the path (see [here](https://github.com/terraform-providers/terraform-provider-openstack/blob/95544ec71a83f5736641955e9d0542951c7f4ca3/openstack/config.go#L80]))

Reply:
i think this should be `cacert_file: "${file(\"%s\")}" % openstack.os_cacert`

Reply:
alternatively, avoid the complication entirely by specifying an absolute path

Reply:
Thanks, will push fix